README.md

```diff
@@ -1,3 +1,99 @@
-# Node.js Express Web Service using standard `node` image
+# Hardening Docker Workloads using Slim CLI
 
-TO-DO
+## The Problem and the Magic 🪄
+Hello folks and if you are reading this, you might have already flashed a gaze at the picture below but before we come to that, let's take a look at the repository and it basically consists of just a simple (but huge) node app and a Dockerfile majorly. Now let's come back to the image (pun intended ofcourse) below again. The left side is what how the original image of our node app looks like and as visible, its huge 995 MB. Now what if we say that you can get the same app's hardened image with size minified to just 89 MB and vulnerabilites reduced by 99.67% and that too in just few simple steps? Sounds magical right? Well, so says the image on your right and that's the magic we are going to talk about today! 
```

Comment:
General tech writing advice: cut the fluff 🙈 

I totally get it - it's your style. But unless you're writing an entertaining-only piece, it's usually a good idea to remove all the words, phrases, and even sentences that can be removed w/o reducing the amount of information you need to convey. It's a pity. But our writing competes with soo many distractions that it just must be concise.

Reply:
Really liked this advice :pleading_face: